internal/acceptance/openstack/baremetal/v1/nodes_test.go

```diff
@@ -219,7 +219,7 @@ func TestNodesVirtualMedia(t *testing.T) {
 
 	client, err := clients.NewBareMetalV1Client()
 	th.AssertNoErr(t, err)
-	client.Microversion = "1.89"
+	client.Microversion = "1.93"
```

Comment:
Although, it does look like you have updated it, so my confidence on that is low.

Reply:
Yes, the issue is driver related, the test is using ipmi but the feature only supports redfish. I'm looking into fixing it.